docs/developers.rst

```diff
@@ -190,3 +190,41 @@ script in an easy to remember way)::
             * ) echo "Please answer 1, 2 q(quit)";;
         esac
     done
+
+Development on a Mac
+--------------------
+
+These instructions describe how to setup a development environment for keylime on a modern copy of OSX.  Right now, these instructions only cover how to get things running with TPM version 1.2.  It may be possible to build and install tpm2-tools and swtpm2 on a mac, but I haven't attempted that yet.  Despite this limitation with TPM 1.2, you can still do a lot of useful development of APIs, and other non-tpm related stuff on a mac.
```

Comment:
Hope its not to much nit-picking, but could you phrase this as non first person. "It may be possible to build and install tpm2-tools and swtpm2 on a mac, **but this is** yet to be verified" - the rest looks great and happy to merge once the above is tweaked.